index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+        <link rel="stylesheet" type="text/css" href="style.css" />
+    </head>
+
+    <body class="container">
+        <div class="header">
+            <div class="logo">
+                <img src="My Logo.png" alt="" class="logo-img" />
+            </div>
+            <div class="logo"></div>
+        </div>
+
+        <div class="first-div">I'm Alexander. Glad to see you</div>
```

Comment:
- [ ] This should be a main header. Kindly use `h1` rather than `div`. That is more accurate.